satellite/gc/piecetracker.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package gc
```

Comment:
Add a `doc.go` file to this new package with a brief description of the package.